themes/openy_themes/openy_rose/css/styles.css

```diff
@@ -4178,14 +4201,41 @@ html.js .branch__updates_queue__button {
 }
 .node--type-blog.node--view-mode-teaser {
   margin: auto;
-  max-width: 373px;
-  width: 100%;
+}
+@media (min-width: 30em) {
+  .node--type-blog.node--view-mode-teaser {
+    max-width: 373px;
+    width: 100%;
+  }
 }
 .node--type-blog.node--view-mode-teaser .inner-wrapper {
-  margin: 15px 0;
+  height: 190px;
+  width: 228px;
```

Comment:
Width 228px is too small.
288px would fit as well.

Reply:
The problem is the slider. I would have to retheme it or come up with an alternate size just for it. The space and arrows make it not fit well. So to keep it simple I kept the same size for both cases.

Reply:
The minimal resolutions that we want to support is iPhone 6 - 375px width.

Reply:
This is at 375px. It is fine for me.

![screen shot 2018-02-01 at 12 22 29 pm](https://user-images.githubusercontent.com/1504038/35692941-a4fa4df0-074a-11e8-8749-1e147d47a5c0.png)

Reply:
if i make it bigger it breaks the slider and then i have to retheme the slider. im trying to avoid that.

Reply:
@david-hernandez It looks too small on the example. And enough white space on the sides to make it larger - 288px should fit without serious modifications.

Reply:
the issue isnt the space for the image. the slider breaks and i have to fix it.

...actually, give me a few minutes. i made some other changes and those may have made it less broken so i might be able to make it bigger.